sdk/core/azure-core-amqp/src/impl/rust_amqp/amqp/session.cpp

```diff
@@ -72,7 +72,7 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
     }
     else
     {
-      return std::numeric_limits<std::uint32_t>::max();
+      return 1;
```

Comment:
Is this supposed to be checked in? That's a big difference.

Reply:
This was a mistake in a previous checkin - the default for both incoming and outgoing windows should have been 1, not uint_max.